app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -735,6 +817,19 @@ class HomeActivityTest {
     return HomeActivity.createHomeActivity(context, profileId)
   }
 
+  private fun isEllipsized() = object : TypeSafeMatcher<View>() {
```

Comment:
is there any reference of StackOverflow or anything for this? if yes, please add here
`// Reference - link`

Reply:
Done